lib/services/s3.js

```diff
@@ -272,6 +272,10 @@ AWS.util.update(AWS.S3.prototype, {
       return true;
     } else if (error && error.code === 'RequestTimeout') {
       return true;
+    } else if (error.code === 'AuthorizationHeaderMalformed' &&
```

Comment:
Just to be safe, can you also add a check that error exists, similar to the conditional before this one?